app/configurator/configurator-state.tsx

```diff
@@ -359,7 +359,11 @@ const getCachedComponents = (
       sourceType: draft.dataSource.type,
       sourceUrl: draft.dataSource.url,
       locale,
-      cubeFilter: { iri: cubeFilter.iri, joinBy: cubeFilter.joinBy },
+      cubeFilter: {
```

Comment:
Should this be renamed cubeOptions ?

Reply:
I added a new issue (#1376) I think it would make sense to rename 👍